core/Archive/DataTableFactory.php

```diff
@@ -397,7 +397,7 @@ private function setSubtables($dataTable, $blobRow, $treeLevel = 0)
             }
 
             $blobName = $dataName . "_" . $sid;
-            if (isset($blobRow[$blobName])) {
+            if (!empty($blobRow[$blobName])) {
```

Comment:
cheers @jajm . Maybe we should simply check like `isset($blobRow[$blobName]) && is_array($blobRow[$blobName])`? I haven't looked in detail but wonder if we would still want to go in the `if` when there is eg an empty array? I guess if it was an empty array, it would simply set an empty subtable which probably isn't wanted so the patch looks good with the `!empty()` check actually. Maybe @diosmosis can confirm?

Reply:
I think it should be ok, blobRow[...] should never be set to false/empty array AFAIK.